File: Server/server/src/main/java/server/model/BooksModel.java
package server.model;

public class BooksModel {
	int id;
	String bookName, authorName, tableName, state, floor;

	public String getState() {
		return state;
	}

	public String getFloor() {
		return floor;
	}

	public void setFloor(String floor) {
		this.floor = floor;
	}

	public void setState(String state) {
		this.state = state;
	}

	public int getId() {
		return id;
	}

	public String getBookName() {
		return bookName;
	}

	public String getAuthorName() {
		return authorName;
	}

	public String getTableName() {
		return tableName;
	}

	public void setId(int id) {
		this.id = id;
	}

	public void setAuthorName(String authorName) {
		this.authorName = authorName;
	}

	public void setBookName(String bookName) {
		this.bookName = bookName;
	}

	public void setTableName(String tableName) {
		this.tableName = tableName;
	}
}

File: android/app/src/main/java/com/example/bar/model/BooksModel.kt
package com.example.bar.model

class BooksModel(
    id: Long,
    bookName : String ,
    authorName: String,
    tableName: String,
    state: String,
    floor: String
) {
    var id: Long? = id
    var bookName: String? = bookName
    var authorName: String? = authorName
    var tableName: String? = tableName
    var state: String? = state
    var f: String? = floor
}